### ISSUE TITLE
uk-london-1 not supported

### ISSUE THREAD

**Issue by gshutler** [oracle/oci-volume-provisioner#70]
If I've followed the thread correctly I believe https://github.com/oracle/oci-go-sdk/pull/72 adds support and that is in [oracle/oci-go-sdk v1.0.0](https://github.com/oracle/oci-go-sdk/releases/tag/v1.0.0).

#### Related error

```
F0322 17:08:53.752715       1 provisioner.go:89] Unable to create volume provisioner client: can not create client, bad region configuration: region named: uk-london-1, not valid
```

**Comment by garthy** [comment]
```
Logs from
oci-volume-provisioner
in
oci-volume-provisioner-d565fc7f5-ls678
F0322 17:08:53.752715       1 provisioner.go:89] Unable to create volume provisioner client: can not create client, bad region configuration: region named: uk-london-1, not valid
```

**Comment by gshutler** [comment]
Looks like #71 (released in 0.5.0) has resolved this.
